run_tpcc_experiment.sh

```diff
@@ -24,46 +25,87 @@ while getopts 'a:d:c:' flag; do
   esac
 done
 
+if [ -z "CRL_USERNAME" ]
+  export CRL_USERNAME=$USER
+fi
+
 echo curdate=$curdate
 
-scriptPaths=()
-
-case "${cloud}" in
-  aws)
-    scriptPaths=($(find ./report-data/$curdate/aws -name "*.sh" |tr '\n' ' '))
-    ;;
-  gce)
-    scriptPaths=($(find ./report-data/$curdate/gce -name "*.sh" |tr '\n' ' '))
-    ;;
-  azure)
-    scriptPaths=($(find ./report-data/$curdate/azure -name "*.sh" |tr '\n' ' '))
-    ;;
-  *)
-    echo "unsupported cloud name"
-    exit 1
-esac
-
-warehousePerVcpuList=( 50 75 100 125 150 )
-
-# Get the randome string with length 6.
-rand_str=$(openssl rand -base64 6)
-session_name="run_tpcc_experiment_${cloud}_${rand_str}"
-set +e
-tmux kill-session -t $session_name
-set -e
-tmux new -s $session_name -d
+
+# The rest of the arguments may contain paths to the scripts to execute.
+# If not specified, attempts to find scripts in the cloud directory.
+shift $((OPTIND - 1 ))
+scriptPaths=("$@")
+
+if [[ ${#scriptPaths[@]} == 0 ]]
+then
+  case "${cloud}" in
+    aws)
+      mapfile -t scriptPaths < <( find ./report-data/$curdate/aws -name "*.sh" )
```

Comment:
I think we can switch back to 
`scriptPaths=($(find ./report-data/$curdate/aws -name "*.sh" |tr '\n' ' '))`, which doesn't require a newer bash